packages/treasury/src/liquidateMinimum.js

```diff
@@ -82,9 +72,13 @@ export async function start(zcf) {
       // await deposited, but we don't need the value. We'll need it to have
       // resolved in both branches, so can't put it in Promise.all.
       await deposited;
-      await E(liqSeat)
-        .getOfferResult()
-        .catch(onSwapOutFail);
+      await E(liqSeat).getOfferResult();
+
+      if (AmountMath.isEqual(inBefore, debtorSeat.getAmountAllocated('In'))) {
+        trace('trying again because nothing was liquidated');
+        await onSwapOutNoTrade();
```

Comment:
This trips a mental [No Nested Await](https://github.com/Agoric/agoric-sdk/wiki/No-Nested-Await) flag. I'm not sure how strongly to enforce that. It's not obvious to me how to avoid it here and I don't see an example in the style docs.

Reply:
I checked that this code path is tested; it is.

Reply:
fixed.